src/components/computed-properties.vue

```diff
@@ -3,7 +3,7 @@
     ref="modal"
     size="lg"
     id="computed-properties"
```

Comment:
Looks like `id="computed-properties"` is being used or can be changed to `calculated-properties`.